ci/Jenkinsfile

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env groovy
+library 'status-jenkins-lib@v1.7.13'
+
+pipeline {
+    agent {
+        label 'linux && x86_64 && qt-5.15.2'
+    }
+
+    environment {
+        SQUISH_DIR = "/opt/squish-runner-7.1-20230222-1555"
+        PYTHONPATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/lib/python:${PYTHONPATH}"
+        LD_LIBRARY_PATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/python3/lib:${LD_LIBRARY_PATH}"
+
+        CREDS = credentials('test-rail-api-devops')
+        TESTRAIL_URL = "https://ethstatus.testrail.net"
+    }
+
+    parameters {
+        string(
+          name: 'BRANCH',
+          description: 'Paste a branch name from test repository.',
+          defaultValue: 'master'
+        )
+        string(
+          name: 'BUILD',
+          description: 'Paste a number of PR that you want to test.',
+          defaultValue: 'PR-'
+        )
+        string(
+          name: 'TEST_NAME',
+          description: 'Paste test name/part of test name to run specific test',
+          defaultValue: ''
+        )
+        string(
+          name: 'TEST_SCOPE',
+          description: 'Paste tag to run specific scope of tests',
+          defaultValue: ''
+        )
+        string(
+          name: 'TESTRAIL_RUN_ID',
+          description: 'Test run ID in Test Rail',
+          defaultValue: ''
+        )
+    }
+
+    options {
+        timestamps()
+        /* Prevent Jenkins jobs from running forever */
+        timeout(time: 120, unit: 'MINUTES')
+        /* manage how many builds we keep */
+        buildDiscarder(logRotator(
+            numToKeepStr: '10',
+            daysToKeepStr: '30',
+            artifactNumToKeepStr: '3',
+        ))
+    }
+
+    stages {
+
+        stage('Download') {
+            steps { script {
+                copyArtifacts(
+                    projectName: "status-desktop/branches/linux/x86_64/package/${params.BUILD}/",
+                    filter: 'pkg/*-x86_64.tar.gz',
+                    selector: lastWithArtifacts(),
+                    target: "${env.WORKSPACE}/tmp"
+                )
+                def pkg_path = "${env.WORKSPACE}/${utils.findFile('tmp/pkg/*-x86_64.tar.gz')}"
+                sh "tar -zxvf '${pkg_path}' -C '${env.WORKSPACE}/tmp'"
+                def app_dir = "${env.WORKSPACE}/${utils.findFile('tmp/*.AppImage')}"
+                env.APP_DIR=app_dir
```

Comment:
```suggestion
                env.APP_DIR = "${env.WORKSPACE}/${utils.findFile('tmp/*.AppImage')}"
```

Reply:
I saved it in a variable to easily debug the code

Reply:
I don't understand what that means. `env.APP_DIR` is also a variable.